openassessment/xblock/test/test_staff.py

```diff
@@ -283,25 +306,37 @@ def test_staff_assess_handler(self, xblock):
         self.submit_staff_assessment(xblock, submission, assessment=TEAM_GOOD_ASSESSMENT)
 
         assessment = teams_api.get_latest_staff_assessment(submission['team_submission_uuid'])
-        self._assert_team_assessment(assessment, submission)
+        self._assert_team_assessment(assessment, submission, self.expected_answer)
+
+    @scenario('data/team_submission.xml', user_id='Bob')
+    def test_staff_assess_handler_regrade(self, xblock):
```

Comment:
So all this already exists functionality-wise, and this ticket was more or less just testing?

Reply:
The ticket was written to implement functionality, but it ended up being done separately because it was coupled to grading.